core/src/test/java/io/grpc/PickFirstLoadBalancer2Test.java

```diff
@@ -71,6 +72,7 @@
 
   private static final Attributes.Key<String> FOO = Attributes.Key.of("foo");
   private Attributes affinity = Attributes.newBuilder().set(FOO, "bar").build();
+  private static final PickSubchannelArgs EMPTY_ARGS = mock(PickSubchannelArgs.class);
```

Comment:
Only if you verify at the end of the test that `EMPTY_ARGS` never had an interaction, it's fine to mock the args without any stubbing.